crates/symbolicator/src/services/symcaches.rs

```diff
@@ -295,11 +290,26 @@ impl SymCacheActor {
 
         match meta {
             Some(handle) => {
+                // TODO: while there is some caching *internally* in the bitcode_svc, the *complete*
+                // fetch request is not cached
```

Comment:
I wonder if this can be further reduced by checking https://docs.rs/symbolic/8.3.0/symbolic/debuginfo/macho/struct.MachObject.html#method.requires_symbolmap first, obviously only for macho symbol files too.  that's probably sufficient rather than introduce an entire new cache which costs a lot of duplicate disk space.

Reply:
I don’t think we have a proper `MachObject` at this point in the code?